### DISCUSSION TITLE
Does specific Kontakt library break AG?

### DISCUSSION THREAD

**Discussion by manatactical** [apohl79/audiogridder#1192] [General]
Hi,

I've found out that some very specific library just doesn't work with AG on Kontakt. It's very odd but I think it's worth sharing it.

![Screen Shot 2023-03-21 at 11 41 04 AM](https://user-images.githubusercontent.com/9537424/226584434-034d5d83-82e2-40f8-ae2d-4a9d300d7477.png)

On that third instance, I've loaded 4-5 different libraries, and one of them breaks AG right away; resulting in a crazy high in DAW's audio load and total silence from AG with that said instance. The library doing that is "Embertone's Mountain Dulcimer". And I'm pretty sure it's not the only one.

Is AG able to trace what Kontakt does at the very beginning of the library load?

**Comment by apohl79** [comment]
> Is AG able to trace what Kontakt does at the very beginning of the library load?

nope unfortunately not.
